### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,13 +45,15 @@ status = "generate"
 
 ### Generation in FFI mode
 
-When you're ready, let's generate the FFI part:
+When you're ready, let's generate the FFI part. In the command we'll execute, `../gir-files` is where we cloned the [gir-files repository](https://github.com/gtk-rs/gir-files). You need to put your Gir file in this repository as well to make your like easier. Then let's run the command:
```
Comment (EPashkin) [inline README.md]: "your LIVE easier" ?
IMHO this better be rephrased to be lesser depend from [gir-files repository], something like "directory with your .gir files".
Also https://github.com/gtk-rs/gir-files already mentioned in introduction

**Comment by EPashkin** [comment]
Thanks, I forgot about "doc" mode.
Just small nit.

**Comment by GuillaumeGomez** [comment]
I also need to precise that the types that you manually bind should be added into the "manual" array inside the `Gir.toml` file.

**Comment by sdroege** [comment]
Or separately if you need to configure something for them (`ref_mode`)

**Comment by GuillaumeGomez** [comment]
@sdroege: Not sure to entire follow what you mean. You're speaking in case they want to extend a type?
@EPashkin: Updated.

**Comment by EPashkin** [comment]
@sdroege Your `ref_mode` addition seems merged automatically without problem.
@GuillaumeGomez 👍 after squash. Please merge it yoursels.

**Comment by GuillaumeGomez** [comment]
I discovered that github provide a "squash and merge" option. So nice!

**Comment by EPashkin** [comment]
But it don't produce merge commit :(

**Comment by GuillaumeGomez** [comment]
Ah. Not so nice then. :(

**Pull request merged**